jssource/src_files/include/javascript/popup_helper.js

```diff
@@ -166,6 +164,47 @@ function send_back(module, id)
 	call_back_function(result_data);
 }
 
+function process_email_return (result_data)
+{
```

Comment:
This function seems to be in the wrong file. This should be in EmailsComposeView.js